src/config/texts.js

```diff
@@ -385,6 +385,10 @@ export const texts = {
       onDeactivate:
         'Soll Matomo Analytics deaktiviert werden? Die Deaktivierung von Matomo Analytics wird mit dem nächsten Neustart der App wirksam.',
       yes: 'Ja'
+    },
+    locationService: {
+      onSystemPermissionMissing:
+        'Um diese Einstellung zu aktivieren, muss zunächst die Berechtigung die Ortungsdienste ihres Gerätes zu verwenden in den Systemeinstellungen erteilt werden.'
```

Comment:
could be simpler and without du/sie...

> Um diese Einstellung zu aktivieren muss zunächst die Berechtigung für Ortungsdienste in den Systemeinstellungen erteilt werden.

like that?

Reply:
changed